inc/ocf_err.h

```diff
@@ -134,6 +134,9 @@ typedef enum {
 
 	/** Cache initialized with wrong cache line size */
 	OCF_ERR_CACHE_LINE_SIZE_MISMATCH,
+
+	/** Invalid operation for cache in passive state. */
+	OCF_ERR_INVALID_OPERATION_CACHE_PASSIVE,
```

Comment:
This is rather lengthy name. How about just `OCF_ERR_CACHE_IS_PASSIVE`? We already have very similarly constructed names, like `OCF_ERR_CACHE_EXIST`.